plugin/trino-oracle/src/test/java/io/trino/plugin/oracle/TestingOracleServer.java

```diff
@@ -45,10 +59,27 @@ public TestingOracleServer()
     {
         super("wnameless/oracle-xe-11g-r2");
 
+        Failsafe.with(CONTAINER_RETRY_POLICY).run(this::createContainer);
```

Comment:
`this` is a container and has a state (created, running, healthy, etc.)
it container gets started, but further setup fails, IMO we should start with a new container

as a preparatory refactor, we should replace `extends OracleContainer` with a field in this class (composition instead of inheritance)

Reply:
if we need `waitUntilContainerStarted` we can create a "shim" subclass ("TestingOracleContainer") that will expose this operation publicly.